docs/2 Architecture/3 Standard Modules.md

```diff
@@ -71,6 +112,62 @@ _Read the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/x/par
 
 Those modules are typically installed on default when using starport. There are a range of modules that are also part of the Cosmos SDK, additionally some other public modules have already reached a major level of usage and acceptance. We will look at more advanced modules in the next Chapter.
 
+
+## Governance
+
+The module enables Cosmos-SDK based blockchain to support an on-chain governance system. In this system, holders of the native staking token of the chain can vote on proposals on a 1 token 1 vote basis. Next is a list of features the module currently supports:
+
+- Proposal submission: Users can submit proposals with a deposit. Once the minimum deposit is reached, proposal enters voting period
+- Vote: Participants can vote on proposals that reached MinDeposit
+- Inheritance and penalties: Delegators inherit their validator's vote if they don't vote themselves.
+- Claiming deposit: Users that deposited on proposals can recover their deposits if the proposal was accepted OR if the proposal never entered voting period.
+
+_Read the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/x/gov/spec/README.md)_
+
+
+## Crisis
+
+The crisis module halts the blockchain under the circumstance that a blockchain invariant is broken. Invariants can be registered with the application during the application initialization process.
+
+_Read the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/x/crisis/spec/README.md)_
+
+
+## Slashing
+
+The slashing module enables Cosmos SDK-based blockchains to disincentivize any attributable action by a protocol-recognized actor with value at stake by penalizing them ("slashing").
+
+Penalties may include, but are not limited to:
+
+Burning some amount of their stake
+Removing their ability to vote on future blocks for a period of time.
+This module will be used by the Cosmos Hub, the first hub in the Cosmos ecosystem.
+
+_Read the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/x/slashing/spec/README.md)_
+
+
+## IBC
+
+The `IBC` module, as Inter-blockchain Communication, enables sending native tokens between blockchains. It is divided by a subset of specifications.
```

Comment:
Sending native tokens between chains is one use case of `IBC`.

In a more general point of view `IBC` allows to relay packets between chains and could be used with any compatible modules between two chains